Parsers/ASimAuthentication/ProductParsers/AuthenticationAADManagedIdentity.yaml

```diff
@@ -0,0 +1,70 @@
+Parser:
+  Title: Azure active directory managed identity authentication
+  Version: '0.1.0'
+  LastUpdated: June 17, 2021
+Product:
+  Name: Azure active directory managed identity signin logs
+Normalization:
+  Schema: Authentication
+  Version: 0.1.0'
+References:
+- Title: ASIM Authentication Schema
+  Link: https://aka.ms/AzSentinelAuthenticationDoc
+- Title: ASIM
+  Link: https:/aka.ms/AzSentinelNormalization
+Description: |
+  This Query Parser maps Azure Active Directory Non Interactive sign in logs (AADNonInteractiveUserSignInLogs) to the Azure Sentinel Information Model authenticaion schema.
```

Comment:
This looks like the same description for NonInteractive, shouldn't this be Managed Identity?